### ISSUE TITLE
PHPUnit\TextUI\ResultPrinter is internal

### ISSUE THREAD

**Issue by KDederichs** [mikeerickson/phpunit-pretty-result-printer#155]
Hey,
I'm getting following deprecation notice using Phpunit 8.4.3:

```
The "PHPUnit\TextUI\ResultPrinter" class is considered internal This class is not covered by the backward compatibility promise for PHPUnit. It may change without further notice. You should not use it from "Codedungeon\PHPUnitPrettyResultPrinter\ResultPrinter80".
```

it would be great if we could get rid of that :)

**Comment by mikeerickson** [comment]
Hmm, not seeing that here.  What version of PHP are you using?

**Comment by KDederichs** [comment]
I'm currently using PHP version 7.3.9 on my development machine.

**Comment by ddebin** [comment]
Same error here. Since PHPUnit 8.0, `PHPUnit\TextUI\ResultPrinter` is `@internal`. cf. https://github.com/sebastianbergmann/phpunit/issues/3236

**Comment by mikeerickson** [comment]
Fixed with 0.27.0 release -- https://github.com/mikeerickson/phpunit-pretty-result-printer/releases/tag/0.27.0
